python/processors/Stop0l_postpost.py

```diff
@@ -194,6 +195,8 @@ def main(args):
     if not isdata:
         ## Major modules for MC
         mods += [
+            btagSFProducer(args.era, algo="deepcsv"),
+            BtagSFWeightProducer(DataDepInputs[dataType][args.era]["bTagEff"], args.sampleName, DeepCSVMediumWP[args.era], isfastsim=isfastsim),
             TopReweightProducer(args.era, args.sampleName, isData=isdata),
             DeepTopProducer(args.era, taggerWorkingDirectory, "JESUp", sampleName=args.sampleName, isFastSim=isfastsim, isData=isdata),
```

Comment:
If we don't expect change from other modules, we don't have to rerun them.

Reply:
I think we can remove all DeepTopProducer modules as we only need to redo the merged top uncertainties which happens in "SoftBDeepAK8SFProducer".  We can also remove "TopReweightProducer"?

Reply:
So we only need DeepTopProducer then? I can work on only running that just want to confirm.

Reply:
If both are updating on the systematic, is it safe to run just these two modules?

Reply:
I updated this technically it runs but events are still being produce on my test file. Please check that the correct top tagger modules are being run. When the file is done I will ping @pastika to have a look at the top variables that we expect to be changed. I will check the btagging to make sure they get updated.

Reply:
Ok, I suspect the TopReweightProducer and baseline won't need to rerun. But please check.